libkodimote/addonsource.h

```diff
@@ -12,6 +12,17 @@ class AddonSource : public Files
     QString title() const;
     KodiModel *enterItem(int index);
 
+    MediaFormat mediaFormat() const {
```

Comment:
Is this necessary? As `AddonSource` extends `Files`, which has the same implementation.